### PR TITLE
feat(pci.k8s.dashboard): update network configuration

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/pci-k8s_add-subnet-managment`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #TAPC-478
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by brofik — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "kubernetes_network_form_none": "Aucun (IPs publiques)",
+  "kubernetes_network_form_description": "Par défaut, vos nœuds de travail disposent d'une IPv4 publique. Si vous choisissez un réseau privé, les IP publiques de ces nœuds seront utilisées exclusivement pour leur administration/le lien avec le control plane Kubernetes, et vos noeuds se verront attribuer une IP sur le vLAN du réseau privé choisi.",
+  "kubernetes_network_form_add": "Créer un réseau privé",
+  "kubernetes_network_form_subnet": "Choisissez un sous-réseau pour votre cluster",
+  "kubernetes_network_form_subnet_description": "Sélectionnez le sous-réseau à utiliser pour votre cluster MKS. Veuillez vous assurer que celui-ci respecte les prérequis réseau du service managé MKS.",
+  "kubernetes_network_form_subnet_link": "Lire la documentation",
+  "kubernetes_network_form_subnet_error_no_gateway_ip": "Le sous-réseau sélectionné n'a pas de gateway IP définie. Attention, cela est necessaire si vous souhaitez utiliser des Load Balancer avec une adresse IP publique, vous pouvez créer/selectionner un autre subnet pour votre cluster ou utiliser les paramètres avancés pour définir un subnet dédié à vos Load Balancer.",
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/translations/Messages_fr_FR.json:19]:
```suggestion
  "kubernetes_network_form_subnet_error_no_gateway_ip": "Le sous-réseau sélectionné n'a pas de gateway IP définie. Attention, cela est nécessaire si vous souhaitez utiliser des Load Balancers avec une adresse IP publique, vous pouvez créer/sélectionner un autre subnet pour votre cluster ou utiliser les paramètres avancés pour définir un subnet dédié à vos Load Balancers.",
```

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "kubernetes_network_form_subnet_error_default": "Une erreur est survenue lors du chargement des sous-réseaux pour ce réseau privé : {{ error }}.",
+  "kubernetes_network_form_load_balancers_subnet": "Choisissez un sous-réseau dédié pour vos Load Balancers",
+  "kubernetes_network_form_load_balancers_subnet_description": "Il vous est possible de définir un sous-réseau spécifique pour vos Load Balancers. Cela est notamment nécessaire si le sous-réseau choisi pour votre cluster ne respecte pas les prérequis nécessaires à la création de Load Balancer managés OVHcloud.",
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/translations/Messages_fr_FR.json:22]:
```suggestion
  "kubernetes_network_form_load_balancers_subnet_description": "Il vous est possible de définir un sous-réseau spécifique pour vos Load Balancers. Cela est notamment nécessaire si le sous-réseau choisi pour votre cluster ne respecte pas les prérequis nécessaires à la création de Load Balancers managés OVHcloud.",
```

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "kubernetes_network_form_gateway_public_description": "Chaque noeud utilisera sa propre interface publique",
+  "kubernetes_network_form_gateway_private": "Interface privée des noeuds",
+  "kubernetes_network_form_gateway_private_description_p1": "À selectionner si vous souhaitez utiliser une passerelle",
+  "kubernetes_network_form_gateway_private_description_p2": "ou Custom tel que configuré dans votre réseau privé",
+  "kubernetes_network_form_gateway_private_description_link": "OVHcloud Manager",
+  "kubernetes_network_form_gateway_vrack_field_ip_placeholder": "Saisir une adresse IP",
+  "kubernetes_network_form_gateway_vrack_field_ip_error_pattern": "Veuillez saisir une adresse IP valide",
+  "kubernetes_network_form_label": "Réseau privé attaché",
+  "kubernetes_network_form_none": "Aucun (IPs publiques)",
+  "kubernetes_network_form_description": "Par défaut, vos nœuds de travail disposent d'une IPv4 publique. Si vous choisissez un réseau privé, les IP publiques de ces nœuds seront utilisées exclusivement pour leur administration/le lien avec le control plane Kubernetes, et vos noeuds se verront attribuer une IP sur le vLAN du réseau privé choisi.",
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/translations/Messages_fr_FR.json:14]: Is it "control plane" or "control panel" ?

Comment (y4nnL) [reply packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/translations/Messages_fr_FR.json:14]: It's control plane 👍

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**